massa-pool-worker/src/tests/scenario.rs

```diff
@@ -114,12 +123,22 @@ fn launch_basic_get_block_operation_execution_mock(
         use ControllerMsg::GetFinalAndCandidateBalance as GetFinal;
         use ControllerMsg::UnexecutedOpsAmong as Unexecuted;
```

Comment:
Not related to these changes but this is a confusing and useless rename

Reply:
I've added a point to the follow up issue #3506 . That sufficient?